pkg/cmd/repo/create/http.go

```diff
@@ -104,6 +109,20 @@ func repoCreate(client *http.Client, hostname string, input repoCreateInput, tem
 		"input": input,
 	}
 
+	if input.GitIgnoreTemplate != "" || input.LicenseTemplate != "" {
+		body := &bytes.Buffer{}
+		enc := json.NewEncoder(body)
+		if err := enc.Encode(input); err != nil {
+			return nil, err
+		}
+		var responseV3 api.RepositoryV3
```

Comment:
`api.repositoryv3` should not be made public. `api.Repository` is our public abstraction. If you need to do new stuff via REST with `repositoryV3` it should be done within the `api` package transforming from `repositoryV3` to `Repository` as needed.